.github/workflows/release.yaml

```diff
@@ -111,6 +113,34 @@ jobs:
         echo "Hash of package should be $ARTIFACT_HASH."
         echo "$ARTIFACT_HASH" | base64 --decode | sha256sum --strict --check --status || exit 1
 
+    - name: Log in to GitHub Container Registry
+      run: docker login ghcr.io --username ${{ github.actor }} --password ${{ secrets.GITHUB_TOKEN }}
+
+    # The Docker image will be built with tags: latest and ${{ RELEASE_TAG }}
+    - name: Build the Docker image
+      run: |
+        RELEASE_TAG=$(git describe --tags --abbrev=0)
+        ./scripts/dev_scripts/build_docker.sh "${IMAGE_NAME}" ${{ github.workspace }} "${RELEASE_TAG}"
```

Comment:
Why not `"$IMAGE_NAME"` and `"$RELEASE_TAG"`?

Reply:
I was following the convention for referencing environment variable in bash commands (`${ENV_NAME}`) so I used it here too. I could switch it to your suggestion to make it consistent.

Reply:
I usually use ShellCheck as a reference, which doesn't seem to recommend it. Yes, it's good to be consistent with other parts of our code.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/d2fe70dc8bc12b52071d18e885bf63b86f9dcc25